cmd/nodepool/create.go

```diff
@@ -28,13 +28,15 @@ func NewCreateCommand() *cobra.Command {
 		ClusterName:  "example",
 		NodeCount:    2,
 		ReleaseImage: "",
+		NodepoolArch: "x86_64",
 	}
 
 	cmd.PersistentFlags().StringVar(&opts.Name, "name", opts.Name, "The name of the NodePool")
 	cmd.PersistentFlags().StringVar(&opts.Namespace, "namespace", opts.Namespace, "The namespace in which to create the NodePool")
 	cmd.PersistentFlags().Int32Var(&opts.NodeCount, "node-count", opts.NodeCount, "The number of nodes to create in the NodePool")
 	cmd.PersistentFlags().StringVar(&opts.ClusterName, "cluster-name", opts.ClusterName, "The name of the HostedCluster nodes in this pool will join")
 	cmd.PersistentFlags().StringVar(&opts.ReleaseImage, "release-image", opts.ReleaseImage, "The release image for nodes. If empty, defaults to the same release image as the HostedCluster.")
+	cmd.PersistentFlags().StringVar(&opts.NodepoolArch, "nodepool-arch", opts.NodepoolArch, "The processor architecture for the NodePool (e.g. aarch64, x86_64)")
```

Comment:
As this is the "nodepool" command, let's name this just "arch".

Reply:
Ok done. Should we rename the flag [here](https://github.com/openshift/hypershift/blob/5dc24f3c244c5b29b1d3da546ec1200818b62639/cmd/cluster/cluster.go#L70) to `arch` as well?

Reply:
yes please, nodepool is the context itself. Otherwise following that line every single variable/flag would need to be prefixed with nodepool.